public/css/tailwind.css

```diff
@@ -640,7 +644,26 @@ video {
   font-weight: 700;
 }
 
+.form-input {
+  border-width: 2px;
+  border-style: solid;
+  --tw-border-opacity: 1;
+  border-color: rgb(3 43 19 / var(--tw-border-opacity));
+  padding-left: 0.25rem;
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  --tw-text-opacity: 1;
+  color: rgb(3 43 19 / var(--tw-text-opacity));
+}
+
+.btn-primary {
+  border-width: 2px;
+  border-style: solid;
+  --tw-border-opacity: 1;
+  border-color: rgb(3 43 19 / var(--tw-border-opacity));
+}
+
```

Comment:
If this is all our own custom styling shouldn't we keep it out of the tailwind.css and put it in our own css file?